tests/integration/test_convolve.py

```diff
@@ -82,6 +88,39 @@ def test_int(shape, filter_shape):
     check_convolve(a, v)
 
 
+def test_dtype():
+    shape = (5,) * 2
+    arr1 = num.random.randint(0, 5, shape)
+    arr2 = num.random.random(shape)
+    out_num = num.convolve(arr1, arr2, mode="same")
+    out_np = np.convolve(arr1, arr2, mode="same")
+    assert allclose(out_num, out_np)
```

Comment:
I don't see how this test relates to dtypes

Reply:
arr1 is generated by randint, arr2 is generated by random, it will do astype

Reply:
I see, maybe add `# int64` and `# float64` at the end of those lines?